src/ui/map.js

```diff
@@ -614,8 +614,8 @@ class Map extends Camera {
      * If the map's current zoom level is lower than the new minimum,
      * the map will zoom to the new minimum.
      *
-     * @param {number | null | undefined} minZoom The minimum zoom level to set (0-24).
-     *   If `null` or `undefined` is provided, the function removes the current minimum zoom (i.e. sets it to 0).
+     * @param {number | null | undefined} minZoom The minimum zoom level to set (-2 - 24).
+     *   If `null` or `undefined` is provided, the function removes the current minimum zoom (i.e. sets it to -2).
```

Comment:
I think this comment should be expanded to clarify that negative zoom levels equate to containers < 512px square. It's not possible to set a map to a negative zoom when the container is larger than a tile.

Reply:
Does fb015e8 look clear enough?